publiccode.yml

```diff
@@ -0,0 +1,89 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+name: 'itowns'
+url: 'https://github.com/iTowns/itowns'
+landingURL: 'https://github.com/iTowns/itowns.git'
+softwareVersion: "2.44.2" # needs to be updated
+releaseDate: '2024-09-17'# needs to be updated
+softwareType: library
+logo: "https://github.com/iTowns/itowns/blob/master/examples/images/itowns_logo.png"
+inputTypes: # ToDo : complete
+  - application/json
+platforms:
+  - web
+categories:
+  - website-builder
+  - data-visualization
+  - it-development
+usedBy: # ToDo : complete and ask for permissions
+  - IGN (https://www.ign.fr/)
+  - Ciril Grp (https://www.ciril.net/fr/)
+  - Sogelink (https://www.sogelink.com/)
+  - Laboratoire Liris (https://liris.cnrs.fr/)
+  - Laboratoire Lastig (https://www.umr-lastig.fr/)
```

Comment:
We could only let IGN and Ciril Grp, known as the the main maintainer. it's will be more easier to maintain the file. Nevertheless I find it interesting to list all the stakeholders of the project.

Reply:
The idea is to list _users_, not just stakeholders.  What's of particular interest for the code.gouv.fr Awesome list is the list of users _from the public sector_, e.g. institution with an id on https://lannuaire.service-public.fr.

I understand it might be difficult to track/maintain, but adding stable users from other public sector organizations here is sufficient.